Ports/qt6-qt5compat/package.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=qt6-qt5compat
+version=6.2.3
+workdir=qt5compat-everywhere-src-${version}
+useconfigure=true
```

Comment:
Please put all of these in single quotes

Reply:
If I understood you right, it should be fixed in the latest commit